### ISSUE TITLE
Update beluga codes to new syntax

### ISSUE THREAD

**Issue by Niols** [Beluga-lang/Meta#1]

**Comment by Niols** [comment]
OK for Smallstep vs bigstep.
Where should we put the codes corresponding to this book?

**Comment by pientka** [comment]
I think we should create a separate directory labelled beluga-code with subdirectories corresponding to each chapter or part. Each chapter should be an individual file as well a config file.  Config files (sources.cfg) which list all the files that must be compiled. 

It might be easiest to do this by hand even if this means the code can become out of sync.

**Comment by Niols** [comment]
I agree, I would prefer to do it by hand instead of extracting it from LaTeX.
We will also have to explain how to see the examples somewhere (in the paper or in the README)
